src/exchangeRateRouter.ts

```diff
@@ -166,6 +168,54 @@ const exchangeRatesCleaner: express.RequestHandler = (req, res, next): void => {
   next()
 }
 
+const queryRedis: express.RequestHandler = async (
```

Comment:
This is more then fast enough for now, but in the future, if we ever decide for some reason to unlimit the amount of pairs and you would be able to call a huge amount, it would be better to optimize it even further to be able to get all pairs at once for all the timestamps. It would require a bit more pre-proccesing but less calls.